oshdb-filter/README.md

```diff
@@ -0,0 +1,108 @@
+oshdb filter
+============
+
+A parser and interpreter for filters which can be applied on OSM entities. It allows, for example, to filter by various combinations of OSM tags.
+
+Before this was integrated in the main OSHDB repository, this was developed as an independent library called [“_ohsome filter_”](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/libs/ohsome-filter).
+
+Usage
+-----
+
+Filters can be used directly in the OSHDB:
+
```

Comment:
I think somewhere has to be mentioned that you should add the dependency. This was here in an older version of the README.